test/integration/apimachinery/watch_restart_test.go

```diff
@@ -259,7 +259,7 @@ func TestWatchRestartsIfTimeoutNotReached(t *testing.T) {
 					t.Fatalf("Watch should have timed out but it exited without an error!")
 				}
 
-				if err != wait.ErrWaitTimeout && tc.succeed {
+				if !wait.Interrupted(err) && tc.succeed {
```

Comment:
thanks
IIUC, `wait.Interrupted` applies only to new `PollUntilContextCancel` and `PollUntilContextTimeout` functions.
Does `wait.Interrupted` work well with `watchtools.UntilWithoutRetry`?

Reply:
I'm very sorry for replying to you so late! 
I could not find any evidence to prove that `wait.Interrupted` can do well with `watchtools.UntilWithoutRetry`.  I updated my pr  :)